examples/databinding/java/com/basicapp/BUILD

```diff
@@ -0,0 +1,23 @@
+load("@rules_android//android:rules.bzl", "android_binary", "android_library")
+
+android_binary(
+    name = "basic_app",
```

Comment:
Since this is a databinding example, it's no longer really a "basic" app. Let's make some naming changes:

1) This directory should be called databinding_app or something similar
2) This app should be called databinding_app or something similar